.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    "env": {
+        "commonjs": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-console": 0,
```

Comment:
if I just wipe out this line, using console gives `unexpected console statement`